permissions-wifi/build.gradle.kts

```diff
@@ -0,0 +1,33 @@
+plugins {
+    alias(libs.plugins.nordic.library.compose)
+    alias(libs.plugins.nordic.hilt)
+    alias(libs.plugins.nordic.nexus)
+}
+group = "no.nordicsemi.android.common"
+
+nordicNexusPublishing {
+    POM_ARTIFACT_ID = "permissions-nfc"
+    POM_NAME = "Nordic library for checking required nfc permission."
```

Comment:
This should be: ...for checking permissions required for scanning for Wi-Fi networks.